### ISSUE TITLE
For those that like dubs

### ISSUE THREAD

**Issue by 0xW1sKy** [TRaSH-/Sonarr-Restrictions-For-Better-Quality-Releases#1]
Can you please make either A) a setting for those of us that desire dubs 
Or B) some guidance on how to use what's already there. 

In my ideal scenario I'd have a configuration I could apply that prefers dubs over anything else. Subs would be acceptable in absence of a dub but ideally upgrades would run until a dub is found.

I recognize this probably isn't the best repo for this issue but I'm not sure which is the correct one.

**Comment by TRaSH-** [comment]
The best place would be Discord, didn't even know this still existed.
Did you take a look at the original guide

**Comment by 0xW1sKy** [comment]
> The best place would be Discord, didn't even know this still existed.
> 
> Did you take a look at the original guide 

Yes I've gone through the guide and now I currently manage all my configs via notifiarr. 

I just grabbed the dubs option for now and made it +10k but I honestly don't know if that's best for the overall scoring system. 

How it interacts with dual audio is also a bit confusing. Mainly just an ask to make it a bit easier for people that actually want the dubs.
